featurex/stimuli/video.py

```diff
@@ -15,7 +16,14 @@ def __init__(self, video, frame_num, filename=None, data=None):
         self.frame_num = frame_num
         self.duration = 1. / video.fps
         self.onset = frame_num * self.duration
-
+    
+class VideoKeyframeStim(VideoFrameStim):
```

Comment:
These frames are meant to represent following similar frames, so maybe "representative frame?"
Other ideas:
- Shot frame
- Scene frame